ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -29,24 +31,32 @@ def attach_context(request):
         service = app[CONFIG_KEY]['service']
         distributed_tracing = app[CONFIG_KEY]['distributed_tracing_enabled']
 
+        context = tracer.context_provider.active()
+
+        # Create a new context based on the propagated information.
+        # Do not fill context with distributed sampling if the tracer is disabled
+        # because the would call the callee to generate references to data which
+        # has never been sent to agent.
+        # [TODO:christian] this is quite generic and applies to any similar library so
+        # at some point we should have some shared code which populates context from headers.
+        if tracer.enabled and distributed_tracing:
+            trace_id = int(request.headers.get(PARENT_TRACE_HEADER_ID, 0))
+            parent_span_id = int(request.headers.get(PARENT_SPAN_HEADER_ID, 0))
+            sampling_priority = request.headers.get(SAMPLING_PRIORITY_HEADER_ID)
+            # keep sampling priority as None if not propagated, to support older client versions on the parent side
+            if sampling_priority:
```

Comment:
This test is wrong it should be `is not None`

Reply:
https://github.com/DataDog/dd-trace-py/pull/359/commits/92ed47f4848194509f6682b7240b19c8883d4b81